### ISSUE TITLE
Raw identifiers in use statements affect the ordering

### ISSUE THREAD

**Issue by ChrisGreenaway** [rust-lang/rustfmt#3791]
For example:

```
use websocket::r#async::futures::Stream;
use websocket::client::ClientBuilder;
use websocket::result::WebSocketError;
```

gets reformatted as:

```
use websocket::client::ClientBuilder;
use websocket::r#async::futures::Stream;
use websocket::result::WebSocketError;
```

**Comment by scampi** [comment]
fixed in `rustfmt-2.0` branch

**Comment by calebcartwright** [comment]
Note that this is still need of being backported, but with the version gates applied
